cmd/kubeadm/app/discovery/token/token.go

```diff
@@ -88,7 +88,7 @@ func RetrieveValidatedClusterInfo(discoveryToken string, tokenAPIServers, rootCA
 		}
 		detachedJWSToken, ok := insecureClusterInfo.Data[bootstrapapi.JWSSignatureKeyPrefix+tokenID]
 		if !ok || len(detachedJWSToken) == 0 {
-			return nil, fmt.Errorf("there is no JWS signed token in the %s ConfigMap. This token id %q is invalid for this cluster, can't connect", bootstrapapi.ConfigMapClusterInfo, tokenID)
+			return nil, fmt.Errorf("token id %q is invalid for this cluster, can't connect or it is expired. Use \"kubeadm token create\" on the master node for creating a new valid token", tokenID)
```

Comment:
small nit: remove ", can't connect" (the token is used for authentication after the connection is already established)

Reply:
Done, thanks